src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -28,6 +28,7 @@ case class WorkspaceName(
                       @(ApiModelProperty@field)(required = true, value = "The name of the workspace")
                       name: String) extends Identifiable {
   def path : String = "workspaces/" + namespace + "/" + name
+  def bucketName : String = Seq("cloud_pilot", namespace, name) mkString "-"
```

Comment:
I would not use cloud_pilot as a prefix, maybe FireCloud if you want to use something. But I would also append a GUID and store the bucket name as a property in the database.